### DISCUSSION TITLE
Welcome to nuxt-vue3-google-signin Discussions!

### DISCUSSION THREAD

**Discussion by kasvith** [wavezync/nuxt-vue3-google-signin#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jarosjan** [comment]
Hi, 
Firstly, I want to say that I love how easy the plugin is to use and love your documentation. I wonder if there is a way to specify scopes. We need to add another scope for google drive, but I cannot see any mention in documentation. 
Can you help us out with that please?

Thanks in advance.

**Reply by kasvith** [reply]
You can add scopes simply with a custom button

**Reply by jarosjan** [reply]
That's what I thought. With useTokenClient I got Bearer token. For our needs it is a requirement to have JWT. With GoogleSignInButton I can get JWT. Can I also get JWT with useTokenClient somehow? 

Thank you for your time. I really appreciate it.

**Comment by NazDov** [comment]
Hi, I love the plugin. Is there a way to have sign in open as a redirect and not inside the popup for better mobile experience? thanks
